### DISCUSSION TITLE
qBittorent v4.5.2. Plugins not supported

### DISCUSSION THREAD

**Discussion by Andrea-Guarneri** [qbittorrent/search-plugins#230] [Q&A]
Hello everyone,

Today I installed the latest version of qBittorent v4.5.2. Then later on when I went to install the plugins previously downloaded with the old version.

They were rejected them, saying it doesn't support any of those in the list released on the GitHub page. 

.... What should I do?   ... Where can I find plugins compatible with the new version?

**Comment by LookHack2023** [comment]
Is the qbittorrent 4.5.2 can not save plugin searching from "All unofficial search plugins"?
Any information to share for solving it?

**Comment by ngosang** [comment]
Maybe this helps => https://github.com/qbittorrent/search-plugins/discussions/235#discussioncomment-5959303
